src/styles/card.ts

```diff
@@ -59,11 +59,15 @@ export const cardImg = style({
 });
 
 export const secondaryCard = style({
+    maxWidth: '400px',
```

Comment:
If you want to remove the underlines when hovering on desktop you can add: `textDecoration: 'none',`

Reply:
funka ikke hos mæ....

Reply:
åja, hmm. Bare dropp det da

Reply:
Gets included in another branch!